src/components/ProjectPeople/ProjectPeople.tsx

```diff
@@ -2,40 +2,62 @@ import React from "react"
 import DeleteIcon from '@mui/icons-material/Delete';
 import EditIcon from '@mui/icons-material/Edit';
 import EmailIcon from '@mui/icons-material/Email';
+import PhoneIcon from '@mui/icons-material/Phone';
+import LinkedInIcon from '@mui/icons-material/LinkedIn';
 import './ProjectPeople.css'
 
 interface PersonDetails {
     name: string,
     involvement: string,
     description: string,
     email: string,
+    phone: string,
+    linkedIn: string,
+    isCurrentMember: boolean, // if person is current member or alumni
     edit?: () => void;
     delete?: () => void;
 }
 
 export const Person: React.FC<PersonDetails> = (props) => {
     return (
-        <div className="card">
+        <div className="person-card">
             <div className="card__header">
-                <h1 className="card__title">{ props.name }</h1>
+                <h1 className="card__title">{props.name}</h1>
                 <div className="card__header__icons">
-                    <EditIcon 
-                        className="icon" 
-                        onClick={props.edit}
-                    />
-                    <h3 className="icon icon--spacer">|</h3>
-                    <DeleteIcon
-                        className="icon" 
-                        onClick={props.delete}
-                    />
-                </div>
+                {(() => {
+                        if (props.isCurrentMember) {
+                            return (
+                                <>
+                                <span id="alumni-tag">Alumni</span>
+                                <h3 className="icon icon--spacer">|</h3>
+                                </>
+                            )
+                        }
+                })()}
+                <EditIcon
+                    className="icon"
+                    onClick={props.edit} />
+                <h3 className="icon icon--spacer">|</h3>
+                <DeleteIcon
+                    className="icon"
+                    onClick={props.delete} />
             </div>
-            <h2 className="card__subtitle card__subtitle--involvement">{ props.involvement }</h2>
-            <p className="card__text">{ props.description }</p>
-            <div className="card__email">
-                <EmailIcon className="icon icon--email" />
-                <a href={`mailto:${props.email}`} className="card__subtitle card__subtitle--email">{ props.email }</a>
+        </div><h2 className="card__subtitle card__subtitle--involvement">{props.involvement}</h2><p className="card__text">{props.description}</p><div className="card__contact">
```

Comment:
Not a blocker, but I believe the code may be easier to read if the code is consistent and every tag is on a separate line like so : 😄 
```suggestion
        </div>
        <h2 className="card__subtitle card__subtitle--involvement">{props.involvement}</h2>
        <p className="card__text">{props.description}</p><div className="card__contact">
```

Reply:
Note taken! I will make this change in the PR for my other ticket related to this. I will merge this PR now.